lib/new.js

```diff
@@ -270,8 +270,13 @@ const getTemplateFromArgs = (args) => {
     // Checks if the flag is paired with a string.
     // (If the flag is provided without the string, it will return `true`.)
     if (typeof args.template === 'string') {
-      name = args.template;
-      url = `${url}-${name}`;
+      if (args.template.indexOf(':') > -1) {
+        name = args.template;
```

Comment:
Non-blocking, but `name = args.template;` could be pulled out of the `if/else` since the statement is the same in both.

Reply:
Good catch @Blackbaud-SteveBrush.  I incorporated your "non-blocking feedback."  :-)